docs/tutorials/image_classification/beginner.md

```diff
@@ -25,7 +25,7 @@ filename = ag.download('https://autogluon.s3.amazonaws.com/datasets/shopee-iet.z
 ag.unzip(filename)
 ```
 
-After the dataset is downloaded, we load it into a [`Dataset`](/api/autogluon.task.html#autogluon.task.ImageClassification.Dataset) object: 
+After the dataset is downloaded, we load it into a :class:`Dataset` object: 
```

Comment:
after checking the rendered website, the :class: sphinx ref is not working.

Reply:
Do you have a suggestion to render it properly?

Reply:
I think the solution would be to revert to the hyperlink and get rid of the "code" style as Sphinx doesn't support hyperlinks in a code style to the best of my knowledge @zhreshold @innixma

I'll make the changes in a few hours